extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/videos/BackblazeVideosImporterTest.java

```diff
@@ -111,7 +111,7 @@ public void testImportVideo() throws Exception {
         UUID jobId = UUID.randomUUID();
 
         VideoModel videoObject =
-                new VideoModel(title, videoUrl, description, encodingFormat, dataId, albumId, false);
+                new VideoModel(title, videoUrl, description, encodingFormat, dataId, albumId, false, null);
```

Comment:
could you make this change for anywhere that you're touching the params on a given model? (so I'm commenting on this line of this file as an example - but if you could make it on each line you're touching _anyway_ in-passing in this PR, that'd be appreciated!)

```suggestion
                new VideoModel(title, videoUrl, description, encodingFormat, dataId, albumId, false /*inTempStore*/, null /*uploadTime*/);
```

Reply:
This seems like an anti-pattern to me and I'd rather not do that. IntelliJ already does this by default without cluttering up the code, e.g. 
<img width="467" alt="image" src="https://user-images.githubusercontent.com/108419085/187717571-c7cd8bb1-d365-4aeb-86b5-f4f308e1dd0f.png">

Reply:
Got it - I'll add this to the list of things to get our codebase consistent/automated on (there's pros/cons to doing it, but I must've been mistaken that folks were agreed on doing it - so perfect thing to punt for some hackathon to find agreement on).